ioos_qc/qartod.py

```diff
@@ -618,7 +621,7 @@ def spike_test(
             flag_arr[i] = QartodFlags.MISSING
 
         # Check if diff is masked but not in inp (this indicates that original data is not missing,
-        # but the data point got masked in diff lines 575-580 due to trying to calculate a value
+        # but the data point got masked in diff lines 575-580 or in lines 582-593 due to trying to calculate a value
```

Comment:
Comments with line numbers will get stale and/or wrong quickly. We need to fix that, but it is beyond the scope of this PR.